controller/index.js

```diff
@@ -370,6 +384,20 @@ var Controller = function (agol, BaseController) {
   }
 
   /**
+   * Stub a route redirect to make testing private functions easier
+   */
+  controller.testRoute = function (req, res) {
+    controller.testMethod(req, res)
+  }
+
+  /**
+   * Stub a method for test route than can be easily wrapped
+   */
+  controller.testMethod = function (req, res) {
+    res.status(418).send('Nothing to see here.')
+  }
```

Comment:
Yeah, I agree this is not so great. But currently the controller relies on a ton of private methods and this is the only way to test them as units. I'd like to eventually refactor most of the private functions out of the controller.